roiextractors/segmentationextractor.py

```diff
@@ -222,7 +252,7 @@ def get_num_rois(self):
         no_rois: int
             integer number of ROIs extracted.
         """
-        return len(self.get_roi_ids())
+        return self._roi_response.shape[0]
```

Comment:
@Saksham20 IMO `len(self.get_roi_ids())` is better. Imagine we make a tool to manually select some ROIs from an image: we could then instantiate the `NumpySegmentationExtractor`, but there would be no roi responses (yet). Instead `self._get_roi_ids()` should always be implemented.